### PR TITLE
Use `TypedArray<Vector3i>` for `GridMap::get_used_cells`

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: modules/gridmap/doc_classes/GridMap.xml [hunk 1]
```diff
@@ -74,7 +74,7 @@
 			</description>
 		</method>
 		<method name="get_used_cells" qualifiers="const">
-			<return type="Array" />
+			<return type="Vector3i[]" />
 			<description>
 				Returns an array of [Vector3] with the non-empty cell coordinates in the grid map.
```
Comment (fire-forge) [inline modules/gridmap/doc_classes/GridMap.xml:79]:
```suggestion
				Returns an array of [Vector3i] with the non-empty cell coordinates in the grid map.
```

**Review by fire-forge — APPROVED** [review]
Looks good aside from the small documentation change 👍

**Comment by KoBeWi** [comment]
This and #60054 are wrong. The return type is array of Vector3i, but the method appends Vector3. Also, as mentioned in the other PR, these 2 PRs should be merged into 1.

**Comment by YuriSizov** [comment]
Superseded by https://github.com/godotengine/godot/pull/64009.

**Pull request closed without merging**